examples/kitchen-sink/gatsby-node.js

```diff
@@ -30,7 +30,10 @@ exports.createPages = ({ graphql, actions }) => {
           createPage({
             path: `/${node.fileNode.sourceInstanceName}/${node.fileNode.name}`,
             component: node.fileAbsolutePath, //blogPost,
-            context: { absPath: node.absolutePath }
+            context: {
+              absPath: node.absolutePath,
+              tableOfContents: node.tableOfContents
```

Comment:
Since this is the kitchen-sink example this is fine, but I want to be careful about adding generated content to the `pageContext` by default in gatsby-mdx itself.